ADOPTERS.md

```diff
@@ -0,0 +1,11 @@
+# Who Is Using Trickster
+
+As the Trickster community grows, we'd like to keep track of who is using it in their stack. We invite you to submit a PR with your company name and @githubhandle to be included on the list.
+
+1. [Comcast](https://comcast.github.io) [[@jranson](https://github.com/jranson)]
+2. [Selfnet e.V.](https://www.selfnet.de/) [[@ThoreKr](https://github.com/ThoreKr)]
+3. [swarmstack](https://github.com/swarmstack) [[@mh720](https://github.com/mh720)]
+4. [Hostinger](https://www.hostinger.com/) [[@ton31337](https://github.com/ton31337)]
+5. [The Remote Company (MailerLite, MailerSend, MailerCheck, YCode)](https://www.remotecompany.com) [[@aorfanos]](https://github.com/aorfanos)
+
+© 2021 The Linux Foundation. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our [Trademark Usage](https://www.linuxfoundation.org/trademark-usage) page.

```

Comment:
This copyright footer actually belongs at the end of the README.md and should not be moved over. Otherwise, looks good, thanks!